src/main/java/com/quinzeminutespourmoi/quinzePourMoi/entities/User.java

```diff
@@ -137,11 +137,14 @@ public void setRole(String role) {
 
     @Override
     public Collection<? extends GrantedAuthority> getAuthorities() {
-       final List<GrantedAuthority> authorities = new ArrayList<GrantedAuthority>();
-       if(getMail().equals(mail)) {
-           authorities.add(new SimpleGrantedAuthority("ADMIN"));
-       }
-       return authorities;
+        final List<GrantedAuthority> authorities = new ArrayList<GrantedAuthority>();
+        if(getMail().equals(mail)) {
```

Comment:
getMail() retourne this.mail, le code dit donc :

    if(this.mail.equals(this.mail))

-> c'est toujours vrai : le if peut sauter

Reply:
la ligne suivante reste par contre :

    authorities.add(new SimpleGrantedAuthority("USER"));

Reply:
ok